src/shapes/textbox.class.js

```diff
@@ -271,23 +271,25 @@
       var wrapped = [], i;
       this.isWrapping = true;
       for (i = 0; i < lines.length; i++) {
-        wrapped = wrapped.concat(this._wrapLine(lines[i], i, desiredWidth));
+        wrapped.push.apply(wrapped, this._wrapLine(lines[i], i, desiredWidth));
       }
       this.isWrapping = false;
       return wrapped;
     },
 
     /**
      * Helper function to measure a string of text, given its lineIndex and charIndex offset
-     * it gets called when charBounds are not available yet.
+     * It gets called when charBounds are not available yet.
+     * Override if necessary
+     * Use with {@link fabric.Textbox#wordSplit}
+     * 
      * @param {CanvasRenderingContext2D} ctx
      * @param {String} text
      * @param {number} lineIndex
      * @param {number} charOffset
      * @returns {number}
-     * @private
      */
-    _measureWord: function(word, lineIndex, charOffset) {
+    measureWord: function(word, lineIndex, charOffset) {
```

Comment:
to be 100% honest with consumer, we can leave it with the low dash, port it to 5.x ( because is a nice change ) and then we can rename it in a single commit later.

Reply:
bb262710